### ISSUE TITLE
error while loading 

### ISSUE THREAD

**Issue by georgiestanaccount** [CGessinger/Creatures-And-Beasts-Minecraft-Mod#47]
[crash-2023-01-01_17.54.56-fml.txt](https://github.com/bonsaistudi0s/Creatures_And_Beasts-Forge/files/10329605/crash-2023-01-01_17.54.56-fml.txt)

![crash](https://user-images.githubusercontent.com/121730678/210186912-328a3f69-e48e-471d-8f4d-eb30ee79d800.PNG)
![crashh](https://user-images.githubusercontent.com/121730678/210186913-6868b116-4ec7-4def-890e-173bd21d2956.PNG)
![crashhhhh](https://user-images.githubusercontent.com/121730678/210186914-c45589e0-7b68-4bc2-8610-b409b5d1e9eb.PNG)

**Comment by SwanX1** [comment]
[infernalstudios/Blazing-Config@3.0.3](https://github.com/infernalstudios/Blazing-Config/compare/3.0.2...3.0.3) has fixed it, just waiting on the config library version for this mod to be updated!
Note: The update won't fix existing configs, it will just fix the issue of it duplicating.

Duplicate of #41

**Comment by Dariensg** [comment]
Resolved in 1.5.0. Delete the duplicate text in your config file and it shouldn't happen again.
